### PR TITLE
Change custom op example to new API and add the doc

### PR DESCRIPTION
**Pull request by hwangdeyu**
fixes https://github.com/onnx/tensorflow-onnx/issues/1852

Signed-off-by: Deyu Huang <deyhuang@microsoft.com>

### CODE REVIEW

File: examples/tf_custom_op/custom_op.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
+<!--- SPDX-License-Identifier: Apache-2.0 -->
```
Comment (hwangdeyu) [inline examples/tf_custom_op/custom_op.md:1]: Hi @EmmaNingMS, do you have any comment for this tf custom op conversion doc?

File: examples/tf_custom_op/double_and_add_one_custom_op.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+from tf2onnx.handler import tf_op
+from tf2onnx.tf_loader import tf_placeholder
+
+
+DIR_PATH = os.path.realpath(os.path.dirname(__file__))
+saved_model_path = os.path.join(DIR_PATH, "model.onnx")
+tf_library_path = os.path.join(DIR_PATH, "double_and_add_one.so")
+
+
+@tf_op("DoubleAndAddOne")
```
Comment (hwangdeyu) [inline examples/tf_custom_op/double_and_add_one_custom_op.py:19]: Implementation of this TF custom op.

File: examples/tf_custom_op/double_and_add_one_custom_op.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+class DoubleAndAddOne:
+    @classmethod
+    def version_1(cls, ctx, node, **kwargs):
+        node.type = "Mul"
+        node_shape = ctx.get_shape(node.input[0])
+        node_dtype = ctx.get_dtype(node.input[0])
+        node_np_dtype = utils.map_onnx_to_numpy_type(node_dtype)
+
+        const_two = ctx.make_const(utils.make_name("cosnt_two"), np.array([2]).astype(node_np_dtype)).output[0]
```
Comment (fatcat-z) [inline examples/tf_custom_op/double_and_add_one_custom_op.py:28]: typo:

const_two

Comment (hwangdeyu) [reply examples/tf_custom_op/double_and_add_one_custom_op.py]: Fixed, thanks!

**Review by fatcat-z — APPROVED** [review]
LGTM, thanks!

**Pull request merged**